include/exadg/convection_diffusion/driver.cpp

```diff
@@ -188,28 +187,154 @@ Driver<dim, Number>::ale_update() const
   time_int_bdf->ale_update();
 }
 
+template<int dim, typename Number>
+void
+Driver<dim, Number>::mark_cells_coarsening_and_refinement(dealii::Triangulation<dim> & tria,
+                                                          VectorType const & solution) const
+{
+  mark_cells_kelly_error_estimator(tria,
+                                   pde_operator->get_dof_handler(),
+                                   pde_operator->get_constraints(),
+                                   *pde_operator->get_mapping(),
+                                   solution,
+                                   application->get_parameters().degree +
+                                     1 /* n_face_quadrature_points */,
+                                   application->get_parameters().amr_data);
+}
+
+template<int dim, typename Number>
+void
+Driver<dim, Number>::setup_after_coarsening_and_refinement()
+{
+  // Update mapping
```

Comment:
I put some assertions here since we do not yet cover (i) ALE mappings (the hierarchy goes `DeformedMappingBase`  is a `MappingDoFVector` is a `MappingQCache`), (ii) Mappings based on `MappingQCache`. So I added a `DeformedCubeManifold` and `mapping_degree = degree` to test the remaining case of a static, but non-affine `mapping` in `rotating_hill/application.h`. Later, we will consider the interpolation of a DoFVector describing the mapping and can hence support the rest here as well!